### DISCUSSION TITLE
Sales & Coupons

### DISCUSSION THREAD

**Discussion by AbdelTM** [gameserverapp/Platform#108] [Suggestions]
Add a feature to creates like for spring sales, black friday sales, etc. And coupons to give discounts

**Comment by mvaessen** [comment]
Hi! Thanks for the suggestion! Regarding the coupons. I generally think of coupons that can be used multiple times. But I can imagine that people would want a single usage coupon as well. Any other uses cases that you can think of?

**Reply by AbdelTM** [reply]
I wouldn't mind single/multiple use coupons, not sure what you need to do in terms of coding

**Reply by mvaessen** [reply]
having single use coupons require more work, because you need to register which account already used a coupon. At this stage im mostly trying to understand the scope of the suggestion and what people are looking for. So we built all the requested features from the start.

**Reply by ImKomplex** [reply]
I think the ability to do a "Summer sale" so everything or selected items have a discount and show like "20% off" next to the original price, and single use coupons (varying rates, and also possibly being able to limit what they can be used on) that Server admins can give out would be a good idea.

**Comment by mvaessen** [comment]
This is an open invitation to everyone: Please add your suggestions for this one.

Based on current poll results this may very well be the next thing GSA works on.

Detailed suggestions are most appreciated ❤️

**Comment by Decollates** [comment]
- The ability to set discounts for token-based web shop packs, not just packs sold for fiat cash.

- Automatic Discounts applied to users with specific roles. As an example, a user with a "Patreon - Tier 1" role will have a different discount compared to a user with a "Patreon - Tier 2" role. This would be for both token-based web packs and cash-based packs.

**Comment by newtarkprojects** [comment]
Coupons that have a set amount available, for example first 10 players to claim coupon gets a discount.

Single use global coupons - first player to claim it, gets it.

**Comment by JXYZP-G** [comment]
I think that coupons need to have the availability as a normal webshop like shopify & sellix ect.

Main functions of the coupons:

- Max number of uses
- Selected products / Product Category / Product Tags where the coupon can be applied on
- Percentage / Fixed amount
- Set Expiration Date 

I had more ideas but i forgot them i will add them as soon as I remember :)

**Comment by Ian-Highlander** [comment]
I would very much like to see the ability to do a sort of "Black Friday" thing where we can just (in one place and not have to do it on every individual item) set a global percentage discount in the shop for a set period of time (say 20% for 24 hours across the board or something). It's something I've wanted to be able to do for my users but we have far too many items configured to be able to do it manually one at a time.

**Comment by mvaessen** [comment]
For additional suggestions, please add to #358
